adam/ontology/phase1_ontology.py

```diff
@@ -66,6 +92,120 @@ def subtype(sub: OntologyNode, _super: OntologyNode) -> None:
 # learner.
 ON = OntologyNode("on")
 subtype(ON, SPATIAL_RELATION)
+PART_OF = OntologyNode("partOf")
+"""
+A relation indicating that one object is part of another object.
+"""
+subtype(PART_OF, RELATION)
+
+SIZE_RELATION = OntologyNode("size-relation")
+subtype(SIZE_RELATION, RELATION)
+
+BIGGER_THAN = OntologyNode("biggerThan")
+"""
+A relation indicating that one object is bigger than another object.
+
+This is a placeholder for a more sophisticated representation of size:
+https://github.com/isi-vista/adam/issues/70
+"""
+subtype(BIGGER_THAN, SIZE_RELATION)
+
+SMALLER_THAN = OntologyNode("smallerThan")
+"""
+A relation indicating that one object is smaller than another object.
+
+This is a placeholder for a more sophisticated representation of size:
+https://github.com/isi-vista/adam/issues/70
+"""
+subtype(SMALLER_THAN, SIZE_RELATION)
+
+
+def bigger_than(obj1: SubObject, obj2: SubObject) -> Tuple[SubObjectRelation, ...]:
+    return (
+        SubObjectRelation(BIGGER_THAN, obj1, obj2),
+        SubObjectRelation(SMALLER_THAN, obj2, obj1),
+    )
+
+
+SUPPORTS = OntologyNode("supports")
+"""
+A relation indicating that  one object provides the force to counteract gravity and prevent another 
+object from falling.
+"""
+subtype(SUPPORTS, SPATIAL_RELATION)
+
+
+def supports(obj1: SubObject, obj2: SubObject) -> Tuple[SubObjectRelation, ...]:
+    """
+    Define a support `SubObjectRelation` for `HierarchicalObjectSchema`
+
+    Args:
+        obj1: The `SubObject` which supports obj2
+        obj2: The `SubObject` being supported
+    Returns:
+        `SubObjectRelation` of relationship type supports
+    """
+    return (SubObjectRelation(SUPPORTS, obj1, obj2),)
+
+
+CONTACTS = OntologyNode("contacts")
+"""
+A symmetric relation indicating that one object touches another.
+"""
+subtype(CONTACTS, SPATIAL_RELATION)
+
+
+def contacts(obj1: SubObject, obj2: SubObject) -> Tuple[SubObjectRelation, ...]:
+    """
+    DSL for constructing a 'contacts' spatial_relationship
+
+    Args:
+        obj1: The `SubObject` which has a reciprocal contacts relationship with
+        obj2: The `SubObject` which contacts obj1
```

Comment:
When referencing arguments in docstrings, the convention is to surround them with asterisks